arch/arm/boot/dts/bcm2708-rpi-b-plus.dts

```diff
@@ -68,6 +73,10 @@
 	clock-frequency = <100000>;
 };
 
+&i2c0mux {
+	pinctrl-0 = <&i2c0_pins>;
+};
+
```

Comment:
bcm283x-rpi-i2c0mux_0_28.dtsi is included which sets
```
&i2c0mux {
	pinctrl-0 = <&i2c0_gpio0>;
	pinctrl-1 = <&i2c0_gpio28>;
};
```
Do we need to duplicate? Or is there a commit missing here?

Reply:
i2c0_pins is a downstream entity so I don't want to refer to it from an upstream file, but I do want to retain the name, hence the apparent duplication.